### PR TITLE
Apply /compare redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1wWos3iWMTCqDbxZBDuSVqvY6ZwYux-goCrX0L0E5Weo/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?node-id=78%3A3960&mode=dev)

## QA

- View the site locally in your web browser at: https://microk8s-io-609.demos.haus/compare
- Check page against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4186

### CODE REVIEW

**Comment by juanruitina** [comment]
Just two very minor comments:

- The "Selecting your lightweight Kubernetes distribution..." text in bold should be a `p` element (styling is correct).
- "Get help with K8s" should be an `h2` (styling is correct).

Other than that, looks good to me, feel free to change to UX+1 when you fix.

I'll check on the little copy question with @Sophie-32 when she's back, don't hold merging waiting for it.

**Comment by petesfrench** [comment]
OOS but nav looks bad with this many items in the navigation and the social media links. Not sure what we should do here @juanruitina 
image

**Comment by lyubomir-popov** [comment]
Isn't a navigation update coming in a separate PR? We moved those buttons on the homepage in the hero, and I'd expect to see the new tag navigation as on c.com

**Comment by petesfrench** [comment]
Also not introduced in your PR, but there is a type here: https://github.com/canonical/microk8s.io/pull/609/files#diff-62c41efb7836497db69fe8762baf6ef4b1ed64ab722893a5ce304519eb6428b2R247

**Comment by mtruj013** [comment]
@petesfrench Yup, like Lyubo said I'm handling the nav (and footer) in a seperate PR. The scope for this one is just the /compare page 🙂

File: templates/compare/index.html [hunk 1]
```diff
@@ -10,204 +10,223 @@
...
-          <td data-heading="minikube" class="u-align--center">
-            <i class="p-icon--success">Yes</i>
+          <td data-heading="minikube">
+            yes
           </td>
         </tr>
       </tbody>
     </table>
+    <div class="u-fixed-width">
+    </div>
```
Comment (petesfrench) [inline templates/compare/index.html:196]:
```suggestion
```

File: templates/compare/index.html [hunk 2]
```diff
@@ -10,204 +10,223 @@
...
-        <ul class="p-list is-split">
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>MicroK8s and Kubernetes resources</h2>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <p class="p-text--small-caps">Webinars</p>
```
Comment (petesfrench) [inline templates/compare/index.html:228]:
```suggestion
        <h3 class="p-text--small-caps">Webinars</h3>
```
I think using Headings here works better from an accessibility standpoint

File: templates/compare/index.html [hunk 3]
```diff
@@ -233,15 +252,9 @@ <h3 class="p-heading--4 p-heading-icon__title">Webinars on demand</h3>
...
-
-    <div class="col-4 p-divider__block">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg" alt="">
-          <h3 class="p-heading--4 p-heading-icon__title">Whitepaper</h3>
-        </div>
+      </div>
+      <div class="p-section--shallow">
+        <p class="p-text--small-caps">Whitepaper</p>
```
Comment (petesfrench) [inline templates/compare/index.html:257]:
```suggestion
        <h3 class="p-text--small-caps">Whitepaper</h3>
```
same here

**Review by petesfrench — APPROVED** [review]
LGTM, just a couple minor changes

**Comment by lyubomir-popov** [comment]
@mtruj013 nice work! Giving plus one with one optional suggestion:
on mobile, I would put horizontal scroll on the table, that's what we discussed with @juanruitina iirc.

image

**Pull request merged**